### PR TITLE
Topology2: sdw-jack-generic: add 24 bit format support for sdw jack

### PR DESCRIPTION
**Pull request by bardliao**
Somehow s24_le format is missed.

### CODE REVIEW

File: tools/topology/topology2/platform/intel/sdw-jack-generic.conf [hunk 1]
```diff
@@ -153,7 +153,7 @@ Object.PCM.pcm [
 
 		Object.PCM.pcm_caps.1 {
 			name "volume playback 0"
-			formats 'S16_LE,S32_LE'
+			formats 'S16_LE,S24_LE,S32_LE'
```
Comment (plbossart) [inline tools/topology/topology2/platform/intel/sdw-jack-generic.conf:156]: ok but the name "volume playback 0"  is probably completely wrong, no? Shouldn't this be fixed as well?

File: tools/topology/topology2/platform/intel/sdw-jack-generic.conf [hunk 2]
```diff
@@ -166,7 +166,7 @@ Object.PCM.pcm [
 
 		Object.PCM.pcm_caps.1 {
 			name "Passthrough Capture 0"
-			formats 'S16_LE,S32_LE'
+			formats 'S16_LE,S24_LE,S32_LE'
```
Comment (plbossart) [inline tools/topology/topology2/platform/intel/sdw-jack-generic.conf:169]: same here, why do we have pcm_caps named "Passthrough Capture 0"

**Comment by bardliao** [comment]
@kv2019i @ranj063 Can we merge this first, and do the cleanup later?

**Review by ranj063 — APPROVED** [review]

**Comment by kv2019i** [comment]
@bardliao Ready to go otherwise, but you need a rebase again.

**Comment by bardliao** [comment]
> @bardliao Ready to go otherwise, but you need a rebase again.

Done

**Pull request merged**